nbconvert/exporters/tests/test_pdf.py

```diff
@@ -37,4 +37,6 @@ def test_export(self):
             (output, resources) = self.exporter_class(latex_count=1).from_filename(newpath)
             self.assertIsInstance(output, bytes)
             assert len(output) > 0
+            # tex file should be cleaned up
+            assert 'notebook.tex' not in os.listdir(td)
```

Comment:
Given that this is also removing the `clean_temp_files` command, I'd appreciate a check that ensured any of the companion files would also be removed.

Reply:
Sure. I updated it to make sure only the initial notebook exists in the directory.